lib/seattleflu/db/cli/command/etl/enrollments.py

```diff
@@ -116,57 +117,83 @@ def etl_enrollments(*, action: str):
                     age             = age(enrollment.document),
                     details         = encounter_details(enrollment.document))
 
-                # Find existing collected samples, or create skeletal sample
-                # records containing just the collection barcode linked back to
-                # this encounter.  Sample manifests generated by the processing
-                # lab will usually be loaded later and fill in the rest of the
-                # sample record.
-                for sample in enrollment.document["sampleCodes"]:
-                    barcode = sample.get("code")
+                # Process all the swabs associated with this encounter
+                for swab in enrollment.document["sampleCodes"]:
+                    barcode = swab.get("code")
 
                     if not barcode:
-                        LOG.warning(f"Skipping collected sample with no barcode")
+                        LOG.warning(f"Skipping collected swab with no barcode")
                         continue
 
-                    # XXX TODO: Stop hardcoding this and handle other types.
-                    # ScannedSelfSwab and ManualSelfSwabbed are kit barcodes,
-                    # not collection barcodes.  TestStrip is an identifier
-                    # UUID, not barcode.
-                    #   - trs, 17 May 2019
-                    if sample["type"] != "ClinicSwab":
-                        LOG.warning(f"Skipping collected sample with unknown type {sample['type']}")
+                    if swab["type"] == "ClinicSwab":
```

Comment:
What is the motivation for renaming `sample` in the `etl enrollments` code to `swab`?

Reply:
I couldn't think of a better word to encompass both samples and kits...

Reply:
Are they fundamentally different? I haven't considered kits before, but I thought samples have been swabs all along.